poll/translations/config.yaml

```diff
@@ -13,6 +13,8 @@ locales:
     - pt_BR  # Portuguese (Brazil)
     - ru  # Russian
     - zh_CN  # Chinese (China)
+    - de  # German
+    - ja  # Japaenese
```

Comment:
@umarmughal824 Japanese is already there in the code `ja_JP`. Is there any reason why we would like to have two version of Japanese?